packages/atlas/src/hooks/useSearchResults.ts

```diff
@@ -43,29 +55,68 @@ export const useSearchResults = ({ searchQuery, limit = 50, videoWhereInput }: S
         isCensored_eq: false,
         ...videoWhereInput,
       },
+      orderBy: [
+        VideoOrderByInput.ReactionsCountDesc,
+        VideoOrderByInput.CommentsCountDesc,
+        VideoOrderByInput.CreatedAtDesc,
+      ],
     },
     {
-      skip: !searchQuery,
-      onError: (error) => SentryLogger.error('Failed to fetch search results', 'SearchResults', error),
+      fetchPolicy: 'cache-and-network',
+      notifyOnNetworkStatusChange: true,
+      skip: !text || !isReady,
+      onError: (error) => SentryLogger.error('Failed to fetch video search results', 'SearchResults', error),
     }
   )
 
-  const getChannelsAndVideos = (loading: boolean, data: SearchQuery['search'] | undefined) => {
-    if (loading || !data) {
-      return { channels: [], videos: [] }
+  const {
+    edges: channelsEdges = [],
+    pageInfo: channelsPageInfo,
+    totalCount: channelsTotalCount,
+    loading: channelsLoading,
+    error: channelsError,
+    fetchMore: fetchMoreChannels,
+    refetch: refetchChannels,
+    networkStatus: channelsNetworkStatus,
+  } = useBasicChannelsConnection(
+    {
+      first,
+      where: {
+        title_contains: text,
+        avatarPhoto: {
+          isAccepted_eq: true,
+        },
+      },
+    },
+    {
+      fetchPolicy: 'cache-and-network',
```

Comment:
@kdembler  Not sure if this solution is correct, let me know

Reply:
I'm not sure what problem you are trying to solve with this (and it's changed since then, right?)

Reply:
I have a problem with `first` parameter. When we fire search query from `SearchBox` for first time, we query videos and channels with `first: 3`. When you press enter, we query videos with `first: 20` but in network tab it stays `3`. There is `first` in `cache.ts` but it seems it doesn't do anything.

Reply:
`first` shouldn't be in cache, we want that to be a single list in the cache. We can sync up on that if you want